Matmil

loordoza opened the pull request.

loordoza committed:
Revert "Delete application.properties"

This reverts commit fe9edc9e06474f44020c748830dcb97d2f9f8fe2.

loordoza committed:
Revert "Delete create.sql"

This reverts commit 8c2c2bd797daec9952934d980c5de549b6643bdd.

loordoza committed:
Merge remote-tracking branch 'origin/matmil' into matmil

# Conflicts:
#	src/main/resources/db/create.sql

loordoza committed:
add 'id' to visits

loordoza committed:
VetPage fix

loordoza committed:
PrescriptionPage initial commit

loordoza committed:
PetPage initial commit

loordoza committed:
UserPage fetching rom endpoint

loordoza committed:
little fix

loordoza committed:
db from main

im-stanly committed:
Delete application.properties

please do not commit the application.properties file

im-stanly commented on src/main/java/com/Clinic/WebApp/repository/VisitsRepository.java:
```diff
@@ -72,7 +72,7 @@ private List<VisitsModel> getVisitesByKind(String kind, Object object){
     private List<VisitsModel> getNextVisits(String dbProperty, int id, String equality){
         return jdbcTemplate.query(GET_VISITS_PROPERTIES_SQL +
-                        " WHERE " + dbProperty + "= ? AND visit_date " + equality,
+                        " WHERE " + dbProperty + "= ? AND visit_date " + equality + " ORDER BU ",
```
" ORDER BU " ?

im-stanly reviewed (commented):
Please make sure that your changes works and that you are uploading only the files that should be upload

loordoza committed:
fast fix :D

loordoza committed:
Merge remote-tracking branch 'origin/matmil' into matmil

im-stanly committed:
Delete create.sql

Please keep views and function at the bottom of the file, otherwise the DB will not work on mac

im-stanly committed:
Merge branch 'main' of https://github.com/im-stanly/Animal-Clinic into matmil
